app/screens/Profile/EditProfileScreen.tsx

```diff
@@ -225,16 +224,25 @@ const EditProfileScreen = () => {
           text: "Não",
           style: "cancel",
         },
-        { text: "Sim", onPress: deleteAccount },
+        { text: "Sim", onPress: () => deleteAccount },
```

Comment:
onPress is receiving a function reference instead of calling it.
Use onPress: deleteAccount or onPress: () => deleteAccount() so the handler actually runs.
```suggestion
        { text: "Sim", onPress: deleteAccount },
```